src/aics-image-viewer/components/App/index.tsx

```diff
@@ -15,23 +15,14 @@ import {
   VolumeFileFormat,
 } from "@aics/volume-viewer";
 
-import {
-  AppProps,
-  ShowControls,
-  GlobalViewerSettings,
-  ViewerSettingUpdater,
-  ViewerSettingChangeHandlers,
-  UseImageEffectType,
-} from "./types";
+import type { AppProps, ShowControls, UseImageEffectType } from "./types";
+import type { ViewerState, ChannelState } from "../ViewerStateProvider/types";
```

Comment:
probably should either put a newline after these imports or else group them with the ones below here that don't come from "shared".  or just remove the newline after the last "shared" one.  I guess the main distinctions are third party/dependencies, things coming from "shared", and everything else?

Reply:
I think prettier has some way of auto-sorting the imports by rules...

Reply:
@ShrimpCryptid added a package to nucmorph-colorizer to do this I believe?

Reply:
Yup, it's `@trivago/prettier-plugin-sort-imports` and `eslint-plugin-simple-import-sort.` Note that they occasionally disagree, though I've mostly configured them to work the same.

Package.json: https://github.com/allen-cell-animated/timelapse-colorizer/blob/main/package.json#L22
Prettier config: https://github.com/allen-cell-animated/timelapse-colorizer/blob/main/.prettierrc
Eslint config: https://github.com/allen-cell-animated/timelapse-colorizer/blob/main/.eslintrc